src/clients/c++/examples/image_client.cc

```diff
@@ -930,11 +940,7 @@ main(int argc, char** argv)
   std::vector<int64_t> shape;
   // Include the batch dimension if required
   if (model_info.max_batch_size_ != 0) {
-    if (batch_size > 0) {
```

Comment:
There is a test above for batch_size > 0 so don't need this.

Reply:
Yes I removed this in this PR itself.

Reply:
Ha, yes. I looked at it backwards.